aten/src/ATen/native/native_functions.yaml

```diff
@@ -2174,6 +2174,10 @@
 
 - func: eye.m(int n, int m, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
 
+- func: eye.nk(int n, *, int? k, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
+
+- func: eye.nmk(int n, int m, int k, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
```

Comment:
Maybe add a comment here that this undocumented overload exists for NumPy compatibility, not Python Array API?